openlibrary/catalog/marc/tests/test_data/bin_expect/collingswood_520aa.json

```diff
@@ -17,6 +17,7 @@
     {
       "personal_name": "Philbrick, W. R.",
       "name": "Philbrick, W. R.",
+      "fuller_name": "(W. Rodman)",
```

Comment:
@tfmorris  Changing this to: 
```
"fuller_name": "Philbrick, W. R. (W. Rodman)",
```

would solve my main concerns I mentioned in https://github.com/internetarchive/openlibrary/pull/7652#discussion_r1151301240

This form of `fuller_name` captures all the context from the original MARC in a way that can't become disassociated.

I still don't think it's as helpful as `$0` / `$1`, but it is meaningful to a human reader, and any automated attempt at parsing the data has the best chance of making use of it with the corresponding complete name. 

It'll never appear as a name on a book, so won't be useful for direct string comparisons, but I can see it providing _some_ help to librarians in this form.

Concatenating "name" + "$q" completes this feature in a way it wasn't before. It took me a while to see a solution to the aspect that was bothering me.

I still think #7724 will be more useful, but I won't object to a new field with the 'full' fuller name.